integration-test/src/test/java/uk/gov/gchq/gaffer/integration/impl/NoAggregationIT.java

```diff
@@ -129,6 +129,7 @@ protected Schema createSchema() {
                 .edge(TestGroups.EDGE,
                         new SchemaEdgeDefinition.Builder()
                                 .source(TestTypes.ID_STRING)
+                                .directed(TestTypes.DIRECTED_EITHER)
```

Comment:
**To highlight this line**
This type definition has no validation function.
So directed can be either True or False.